src/models/tasks.py

```diff
@@ -0,0 +1,64 @@
+from transformers import BertConfig, BertForSequenceClassification
+import torch
+from torch import nn
+from loguru import logger
+from torch.nn import CTCLoss
+
+from src.models.vtr.ocr import OCRHead
+from src.utils.common import PositionalEncoding, compute_ctc_loss
+from src.models.embedders.vtr import VTREmbedder
+
+
+class SequenceClassifier(nn.Module):
```

Comment:
Is it the same module? Perhaps, you need to update your branch with master branch